package.json

```
@@ -1,11 +1,12 @@
 {
   "name": "@hirespace/hs-cache-manager",
-  "version": "1.0.3",
+  "version": "1.0.5",
```

This is a feature so needs to be a `minor` version bump.
```suggestion
  "version": "1.1.0",
```